workers_manager.py

```diff
@@ -100,6 +110,33 @@ def _pip_install_helper(package_names):
     for package in package_names:
       pip_main(['install', '-q', package])
 
+  def _poll_wrapper(self, worker_obj, client, userdata, c):
+    if c.payload.decode('utf-8') != '':
+      return
+    poll_device_supported = 'poll_device' in worker_obj.status_update.__code__.co_varnames
+    topic_prefixes = []
+    topic_prefixes.append(userdata.get('global_topic_prefix', ''))
```

Comment:
Please use some topic helpers, to be sure it wont break after some refactoring. https://github.com/zewelor/bt-mqtt-gateway/blob/54ae51814e8ee23c443755485b1a38376e136fe5/mqtt.py#L81

https://github.com/zewelor/bt-mqtt-gateway/blob/54ae51814e8ee23c443755485b1a38376e136fe5/workers/base.py#L10